2022-07-28 18:52:48,113 (44454/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2022-07-28 18:52:48,114 (44454/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2022-07-28 18:56:27,775 (45313/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2022-07-28 18:56:27,776 (45313/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
2022-07-28 19:06:10,754 (45313/MainThread) pgcli.pgexecute ERROR - sql: 'select published_at + interval random()  from item_data limit 10', error: SyntaxError('syntax error at or near "("\nLINE 1: select published_at + interval random()  from item_data limi...\n                                             ^\n')
2022-07-28 19:06:10,757 (45313/MainThread) pgcli.pgexecute ERROR - traceback: 'Traceback (most recent call last):\n  File "/usr/local/Cellar/pgcli/3.4.1_1/libexec/lib/python3.10/site-packages/pgcli/pgexecute.py", line 480, in run\n    yield self.execute_normal_sql(sql) + (sql, True, False)\n  File "/usr/local/Cellar/pgcli/3.4.1_1/libexec/lib/python3.10/site-packages/pgcli/pgexecute.py", line 515, in execute_normal_sql\n    cur.execute(split_sql)\n  File "/usr/local/Cellar/pgcli/3.4.1_1/libexec/lib/python3.10/site-packages/pgcli/pgexecute.py", line 175, in execute\n    psycopg2.extensions.cursor.execute(self, sql, args)\npsycopg2.errors.SyntaxError: syntax error at or near "("\nLINE 1: select published_at + interval random()  from item_data limi...\n                                             ^\n\n'
2022-07-28 19:08:15,190 (45313/MainThread) pgcli.pgexecute ERROR - sql: 'select published_at + random() * (curent_timestamp() - published_at)  from item_data limit 10', error: UndefinedFunction('function curent_timestamp() does not exist\nLINE 1: select published_at + random() * (curent_timestamp() - publi...\n                                          ^\nHINT:  No function matches the given name and argument types. You might need to add explicit type casts.\n')
2022-07-28 19:08:15,191 (45313/MainThread) pgcli.pgexecute ERROR - traceback: 'Traceback (most recent call last):\n  File "/usr/local/Cellar/pgcli/3.4.1_1/libexec/lib/python3.10/site-packages/pgcli/pgexecute.py", line 480, in run\n    yield self.execute_normal_sql(sql) + (sql, True, False)\n  File "/usr/local/Cellar/pgcli/3.4.1_1/libexec/lib/python3.10/site-packages/pgcli/pgexecute.py", line 515, in execute_normal_sql\n    cur.execute(split_sql)\n  File "/usr/local/Cellar/pgcli/3.4.1_1/libexec/lib/python3.10/site-packages/pgcli/pgexecute.py", line 175, in execute\n    psycopg2.extensions.cursor.execute(self, sql, args)\npsycopg2.errors.UndefinedFunction: function curent_timestamp() does not exist\nLINE 1: select published_at + random() * (curent_timestamp() - publi...\n                                          ^\nHINT:  No function matches the given name and argument types. You might need to add explicit type casts.\n\n'
2022-07-28 19:08:21,485 (45313/MainThread) pgcli.pgexecute ERROR - sql: 'select published_at + random() * (curent_timestamp - published_at)  from item_data limit 10', error: UndefinedColumn('column "curent_timestamp" does not exist\nLINE 1: select published_at + random() * (curent_timestamp - publish...\n                                          ^\n')
2022-07-28 19:08:21,485 (45313/MainThread) pgcli.pgexecute ERROR - traceback: 'Traceback (most recent call last):\n  File "/usr/local/Cellar/pgcli/3.4.1_1/libexec/lib/python3.10/site-packages/pgcli/pgexecute.py", line 480, in run\n    yield self.execute_normal_sql(sql) + (sql, True, False)\n  File "/usr/local/Cellar/pgcli/3.4.1_1/libexec/lib/python3.10/site-packages/pgcli/pgexecute.py", line 515, in execute_normal_sql\n    cur.execute(split_sql)\n  File "/usr/local/Cellar/pgcli/3.4.1_1/libexec/lib/python3.10/site-packages/pgcli/pgexecute.py", line 175, in execute\n    psycopg2.extensions.cursor.execute(self, sql, args)\npsycopg2.errors.UndefinedColumn: column "curent_timestamp" does not exist\nLINE 1: select published_at + random() * (curent_timestamp - publish...\n                                          ^\n\n'
2022-07-28 19:08:39,351 (45313/MainThread) pgcli.pgexecute ERROR - sql: 'select published_at + random() * (curent_timestamp() - published_at)  from item_data limit 10', error: UndefinedFunction('function curent_timestamp() does not exist\nLINE 1: select published_at + random() * (curent_timestamp() - publi...\n                                          ^\nHINT:  No function matches the given name and argument types. You might need to add explicit type casts.\n')
2022-07-28 19:08:39,351 (45313/MainThread) pgcli.pgexecute ERROR - traceback: 'Traceback (most recent call last):\n  File "/usr/local/Cellar/pgcli/3.4.1_1/libexec/lib/python3.10/site-packages/pgcli/pgexecute.py", line 480, in run\n    yield self.execute_normal_sql(sql) + (sql, True, False)\n  File "/usr/local/Cellar/pgcli/3.4.1_1/libexec/lib/python3.10/site-packages/pgcli/pgexecute.py", line 515, in execute_normal_sql\n    cur.execute(split_sql)\n  File "/usr/local/Cellar/pgcli/3.4.1_1/libexec/lib/python3.10/site-packages/pgcli/pgexecute.py", line 175, in execute\n    psycopg2.extensions.cursor.execute(self, sql, args)\npsycopg2.errors.UndefinedFunction: function curent_timestamp() does not exist\nLINE 1: select published_at + random() * (curent_timestamp() - publi...\n                                          ^\nHINT:  No function matches the given name and argument types. You might need to add explicit type casts.\n\n'
2022-07-28 19:08:48,496 (45313/MainThread) pgcli.pgexecute ERROR - sql: 'select published_at + random() * (current_timestamp() - published_at)  from item_data limit 10', error: SyntaxError('syntax error at or near ")"\nLINE 1: ...lect published_at + random() * (current_timestamp() - publis...\n                                                             ^\n')
2022-07-28 19:08:48,496 (45313/MainThread) pgcli.pgexecute ERROR - traceback: 'Traceback (most recent call last):\n  File "/usr/local/Cellar/pgcli/3.4.1_1/libexec/lib/python3.10/site-packages/pgcli/pgexecute.py", line 480, in run\n    yield self.execute_normal_sql(sql) + (sql, True, False)\n  File "/usr/local/Cellar/pgcli/3.4.1_1/libexec/lib/python3.10/site-packages/pgcli/pgexecute.py", line 515, in execute_normal_sql\n    cur.execute(split_sql)\n  File "/usr/local/Cellar/pgcli/3.4.1_1/libexec/lib/python3.10/site-packages/pgcli/pgexecute.py", line 175, in execute\n    psycopg2.extensions.cursor.execute(self, sql, args)\npsycopg2.errors.SyntaxError: syntax error at or near ")"\nLINE 1: ...lect published_at + random() * (current_timestamp() - publis...\n                                                             ^\n\n'
2022-07-29 18:25:10,022 (45313/MainThread) pgcli.pgexecute ERROR - sql: "select url from item_dat where is_actual = true and platform = 'meduza'", error: UndefinedTable('relation "item_dat" does not exist\nLINE 1: select url from item_dat where is_actual = true and platform...\n                        ^\n')
2022-07-29 18:25:10,027 (45313/MainThread) pgcli.pgexecute ERROR - traceback: 'Traceback (most recent call last):\n  File "/usr/local/Cellar/pgcli/3.4.1_1/libexec/lib/python3.10/site-packages/pgcli/pgexecute.py", line 480, in run\n    yield self.execute_normal_sql(sql) + (sql, True, False)\n  File "/usr/local/Cellar/pgcli/3.4.1_1/libexec/lib/python3.10/site-packages/pgcli/pgexecute.py", line 515, in execute_normal_sql\n    cur.execute(split_sql)\n  File "/usr/local/Cellar/pgcli/3.4.1_1/libexec/lib/python3.10/site-packages/pgcli/pgexecute.py", line 175, in execute\n    psycopg2.extensions.cursor.execute(self, sql, args)\npsycopg2.errors.UndefinedTable: relation "item_dat" does not exist\nLINE 1: select url from item_dat where is_actual = true and platform...\n                        ^\n\n'
2022-07-29 18:25:16,969 (45313/MainThread) pgcli.pgexecute ERROR - sql: "select url from item_dat where is_actual = true and platform = 'meduza'", error: UndefinedTable('relation "item_dat" does not exist\nLINE 1: select url from item_dat where is_actual = true and platform...\n                        ^\n')
2022-07-29 18:25:16,969 (45313/MainThread) pgcli.pgexecute ERROR - traceback: 'Traceback (most recent call last):\n  File "/usr/local/Cellar/pgcli/3.4.1_1/libexec/lib/python3.10/site-packages/pgcli/pgexecute.py", line 480, in run\n    yield self.execute_normal_sql(sql) + (sql, True, False)\n  File "/usr/local/Cellar/pgcli/3.4.1_1/libexec/lib/python3.10/site-packages/pgcli/pgexecute.py", line 515, in execute_normal_sql\n    cur.execute(split_sql)\n  File "/usr/local/Cellar/pgcli/3.4.1_1/libexec/lib/python3.10/site-packages/pgcli/pgexecute.py", line 175, in execute\n    psycopg2.extensions.cursor.execute(self, sql, args)\npsycopg2.errors.UndefinedTable: relation "item_dat" does not exist\nLINE 1: select url from item_dat where is_actual = true and platform...\n                        ^\n\n'
2022-07-29 18:27:29,137 (45313/MainThread) pgcli.pgexecute ERROR - sql: 'select min(load_date) from data_raw', error: UndefinedColumn('column "load_date" does not exist\nLINE 1: select min(load_date) from data_raw\n                   ^\nHINT:  Perhaps you meant to reference the column "data_raw.loaded_at".\n')
2022-07-29 18:27:29,138 (45313/MainThread) pgcli.pgexecute ERROR - traceback: 'Traceback (most recent call last):\n  File "/usr/local/Cellar/pgcli/3.4.1_1/libexec/lib/python3.10/site-packages/pgcli/pgexecute.py", line 480, in run\n    yield self.execute_normal_sql(sql) + (sql, True, False)\n  File "/usr/local/Cellar/pgcli/3.4.1_1/libexec/lib/python3.10/site-packages/pgcli/pgexecute.py", line 515, in execute_normal_sql\n    cur.execute(split_sql)\n  File "/usr/local/Cellar/pgcli/3.4.1_1/libexec/lib/python3.10/site-packages/pgcli/pgexecute.py", line 175, in execute\n    psycopg2.extensions.cursor.execute(self, sql, args)\npsycopg2.errors.UndefinedColumn: column "load_date" does not exist\nLINE 1: select min(load_date) from data_raw\n                   ^\nHINT:  Perhaps you meant to reference the column "data_raw.loaded_at".\n\n'
2022-07-29 18:27:36,643 (45313/MainThread) pgcli.pgexecute ERROR - sql: 'select min(loaded_date) from data_raw', error: UndefinedColumn('column "loaded_date" does not exist\nLINE 1: select min(loaded_date) from data_raw\n                   ^\nHINT:  Perhaps you meant to reference the column "data_raw.loaded_at".\n')
2022-07-29 18:27:36,643 (45313/MainThread) pgcli.pgexecute ERROR - traceback: 'Traceback (most recent call last):\n  File "/usr/local/Cellar/pgcli/3.4.1_1/libexec/lib/python3.10/site-packages/pgcli/pgexecute.py", line 480, in run\n    yield self.execute_normal_sql(sql) + (sql, True, False)\n  File "/usr/local/Cellar/pgcli/3.4.1_1/libexec/lib/python3.10/site-packages/pgcli/pgexecute.py", line 515, in execute_normal_sql\n    cur.execute(split_sql)\n  File "/usr/local/Cellar/pgcli/3.4.1_1/libexec/lib/python3.10/site-packages/pgcli/pgexecute.py", line 175, in execute\n    psycopg2.extensions.cursor.execute(self, sql, args)\npsycopg2.errors.UndefinedColumn: column "loaded_date" does not exist\nLINE 1: select min(loaded_date) from data_raw\n                   ^\nHINT:  Perhaps you meant to reference the column "data_raw.loaded_at".\n\n'
2022-07-29 18:45:54,740 (45313/MainThread) pgcli.pgexecute ERROR - sql: 'select platform, max(version) group by platform', error: UndefinedColumn('column "platform" does not exist\nLINE 1: select platform, max(version) group by platform\n               ^\n')
2022-07-29 18:45:54,740 (45313/MainThread) pgcli.pgexecute ERROR - traceback: 'Traceback (most recent call last):\n  File "/usr/local/Cellar/pgcli/3.4.1_1/libexec/lib/python3.10/site-packages/pgcli/pgexecute.py", line 480, in run\n    yield self.execute_normal_sql(sql) + (sql, True, False)\n  File "/usr/local/Cellar/pgcli/3.4.1_1/libexec/lib/python3.10/site-packages/pgcli/pgexecute.py", line 515, in execute_normal_sql\n    cur.execute(split_sql)\n  File "/usr/local/Cellar/pgcli/3.4.1_1/libexec/lib/python3.10/site-packages/pgcli/pgexecute.py", line 175, in execute\n    psycopg2.extensions.cursor.execute(self, sql, args)\npsycopg2.errors.UndefinedColumn: column "platform" does not exist\nLINE 1: select platform, max(version) group by platform\n               ^\n\n'
2022-07-29 19:17:24,889 (45313/MainThread) pgcli.pgexecute ERROR - sql: 'select count(*) from trackingl', error: UndefinedTable('relation "trackingl" does not exist\nLINE 1: select count(*) from trackingl\n                             ^\n')
2022-07-29 19:17:24,889 (45313/MainThread) pgcli.pgexecute ERROR - traceback: 'Traceback (most recent call last):\n  File "/usr/local/Cellar/pgcli/3.4.1_1/libexec/lib/python3.10/site-packages/pgcli/pgexecute.py", line 480, in run\n    yield self.execute_normal_sql(sql) + (sql, True, False)\n  File "/usr/local/Cellar/pgcli/3.4.1_1/libexec/lib/python3.10/site-packages/pgcli/pgexecute.py", line 515, in execute_normal_sql\n    cur.execute(split_sql)\n  File "/usr/local/Cellar/pgcli/3.4.1_1/libexec/lib/python3.10/site-packages/pgcli/pgexecute.py", line 175, in execute\n    psycopg2.extensions.cursor.execute(self, sql, args)\npsycopg2.errors.UndefinedTable: relation "trackingl" does not exist\nLINE 1: select count(*) from trackingl\n                             ^\n\n'
2022-07-29 20:48:44,014 (54677/MainThread) pgcli.main INFO - Default pager found in PAGER environment variable: "less"
2022-07-29 20:48:44,014 (54677/MainThread) pgcli.main WARNING - import keyring failed: ModuleNotFoundError("No module named 'keyring'").
